framework/SupervisedLearning/DynamicModeDecomposition.py

```diff
@@ -0,0 +1,319 @@
+
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  Created on May 8, 2018
```

Comment:
Hey Paul,
Can you report here that these files comes from the split of the original SupervisedLearning.py module and author and date (if retrievable) of the initial implementation? ( just to keep track of the "maturity" of the "ROM")